pkg/codegen/codegen_test.go

```diff
@@ -153,6 +153,16 @@ type GetTestByNameResponse struct {
 	JSONDefault  *Error
 }`)
 
+	// Check that types for the application/didcomm-plaintext+json were generated.
+	assert.Contains(t, code, `
+type CreateTestByNameResponse struct {
+	Body         []byte
+	HTTPResponse *http.Response
+	JSON200      *[]Test
+}`)
+	assert.Contains(t, code, "type CreateTestByNameJSONBody Test")
+	assert.Contains(t, code, "type CreateTestByNameJSONRequestBody CreateTestByNameJSONBody")
+
```

Comment:
this covers the part that generates response and request for the didcomm-plain+json